src/game/models/mod.rs

```diff
@@ -2,156 +2,16 @@
 
 use std::f64;
```

Comment:
I wonder if it's because we use `use std::f64::consts::PI;` below. This feels bug report-worthy to me.